src/aatoolbox/datasources/usgs/usgs_ndvi.py

```diff
@@ -0,0 +1,924 @@
+        Country configuration
+    coverage_area : str
+        Area of coverage
+    data_type : str
+        Data type
+    start_date : Union[date, str, Tuple[int], None]
+        Start date. Can be passed as a ``datetime.date``
+        object and the relevant dekad will be determined,
+        as a date string in ISO8601 format, or as a
+        year-dekad tuple, i.e. (2020, 1).
+    end_date : Union[date, str, Tuple[int], None]
+        End date. Can be passed as a ``datetime.date``
+        object and the relevant dekad will be determined,
+        as a date string in ISO8601 format, or as a
+        year-dekad tuple, i.e. (2020, 1).
+    processed_file_suffix : str
+        Suffix for processed file.
+    """
+
+    def __init__(
+        self,
+        country_config: CountryConfig,
+        coverage_area: _VALID_AREAS,
+        data_type: _VALID_TYPES,
+        start_date: Union[date, str, Tuple[int], None] = None,
+        end_date: Union[date, str, Tuple[int], None] = None,
+        processed_file_suffix: str = "",
+    ):
+        super().__init__(
+            country_config=country_config,
+            datasource_base_dir="usgs_ndvi",
+            is_public=True,
+            is_global_raw=True,
+        )
+        self.coverage_area = coverage_area
+
+        # set data type directly in init because should not be
+        # changed in an instance
+        valid_types = _VALID_TYPES.__args__  # type: ignore
+        if data_type not in valid_types:
+            raise ValueError(
+                "`data_type` is not a valid string value. "
+                f"It must be one of {*valid_types,}"
+            )
+        self._data_type = data_type
+
+        # url code and file suffix
+        loc_data_types = {
+            "smoothed": ("temporallysmoothedndvi", ""),
+            "percent_median": ("percentofmedian", "pct"),
+            "median_anomaly": ("mediananomaly", "stmdn"),
+            "difference": ("differencepreviousyear", "dif"),
+        }
+        data_type_values = loc_data_types[data_type]
+        self._data_type_url = data_type_values[0]
+        self._data_type_suffix = data_type_values[1]
+
+        # set dates for data download and processing
+        if start_date is None:
+            # earliest dekad of data is 2002-19
+            self._start_year, self._start_dekad = 2002, 19
+        elif isinstance(start_date, (str, date)):
+            self._start_year, self._start_dekad = _date_to_dekad(start_date)
+        else:
+            self._start_year, self._start_dekad = cast(list, start_date)
+
+        if end_date is None:
+            self._end_year, self._end_dekad = _date_to_dekad(date.today())
+        elif isinstance(start_date, (str, date)):
+            self._end_year, self._end_dekad = _date_to_dekad(end_date)
+        else:
+            self._end_year, self._end_dekad = cast(list, end_date)
+
+        # set processed file suffix
+        self.processed_file_suffix = processed_file_suffix
```

Comment:
This is a public property. Added it because needed a way for users to specify the file suffix so they can `process()` to different levels of aggregation.

Reply:
Oops you're right sorry!

Reply:
Removing anyway based on other comments about the suffix and instead based on aggregation.

Reply:
Won't let me resolve this lol.